public/css/styles.css

```diff
@@ -4,6 +4,10 @@
   margin-right: auto ;
 }
 
+.clickable {
+  cursor: pointer;
+}
+
```

Comment:
makes the cursor look like a pointer.  Cursor looks like a pointer on most buttons by default, but sometimes it doesn't work so we can manually make that happen by adding the class "clickable"